src/en/space-station-14/round-flow/proposals/time-agent.md

```diff
@@ -0,0 +1,83 @@
+
+#### Assist the Past:
+Teleports the agent into a projection of the past - a temporary, empty dimension with featureless figures representing any mobs that were nearby at the time - for however many seconds the unclosed loop had lasted, where all performed actions had already happened before. The damage and other effects that the agent's future self experienced when summoned in reality will be dealt during the assistance in the projection. The center of the temporary dimension’s floor glows the most, and is where **Summon Future Assistance** was targeted originally.
+
+#### Temporal Loop Memories:
+The agent recalls the actions they must perform with **Assist the Past** to close the time loops created by **Summon Future Assistance**.
+
+#### United Synchronization:
+Can be cycled through freely. Multiplied uses of abilities incur multiplied amounts of possibilities, as is expected.
+* **None**: The default.
+* **Past Only**: Using **Summon Past Assistance** uses the ability twice if there are 2 or more **Assist the Future** loops currently prepared for closure.
+* **Past & Future**: Using **Summon Past Assistance** uses the ability as many times as there is **Assist the Future** loops currently prepared for closure. If the amount of prepared **Assist the Future** loops is less than 3, **Summon Future Assistance** is used however many times needed to bring the total number of summoned doppelgangers up to 3. Using **Summon Future Assistance** uses the ability thrice.
+* **Future Only**: Using **Summon Future Assistance** uses the ability twice.
+
+#### Rewind Bubble:
+Expends 4 possibilities. When blown, automatically pops in 21 seconds if not popped manually. (manual popping is simply done by using the ability again while a bubble is currently active)
+* **Blow**: Saves the state of the area in a 5 tile radius around the agent, as well as the agent's position.
+* **Pop**: Rewinds the saved area, as well as any items or mobs tagged/logged by the bubble, to the state everything was in when the bubble was blown. The agent's position, obviously, gets rewinded to their saved position as well.
+
+## Objectives
+#### Nudge Item: 
+* Place Item on a Certain Table: Awards 1 possibility. Can be optional.
+* Place Item in a Certain Box/Bag/Locker: Awards 1 possibility. Can be optional.
+* Ensure Item is in Inventory Until Certain Time: Awards 1-2 possibilities. Can be optional.
+* Ensure Item is in Certain Crewmember's Inventory at Certain Time: Awards 2-3 possibilities. Can be optional.
+
+#### Nudge Life:
+* Ensure Certain Crewmember is Alive at Certain Time: Awards 2-4 possibilities. Revokes 2-4 possibilities if failed.
+* Ensure Certain Crewmember is Dead at Certain Time: Awards 3-5 possibilities. Revokes 3-5 possibilities if failed.

```

Comment:
these are good objectives for an antag like this, i like it

Reply:
Thanks! I think the most difficult one to code would be the 'place item on a certain table' objective, though I suppose it can be done by doing a check every time the agent places any item and just seeing if it's the correct item, as well as if its new location has the same coordinates as the correct table's.